src/status_im/ui/screens/chat/message/message.cljs

```diff
@@ -195,6 +195,12 @@
    [react/image {:style {:margin 10 :width 140 :height 140}
                  :source {:uri (contenthash/url (:hash content))}}]])
 
+(defmethod message-content constants/content-type-image
+  [wrapper {:keys [content] :as message}]
+  [wrapper message
+   [react/image {:style {:margin-vertical 10 :width 140 :height 140 :border-radius 8}
+                 :source {:uri (str "https://ipfs.infura.io/ipfs/" (:hash content))}}]])
```

Comment:
I would not impose IPFS in the URL, we can still validate the link client side if we want to have only IPFS domains, but otherwise changing this would be breaking. Rather I would pass the full url, and then we can only show if `starts-with` ipfs or something.
Also markdown supports images, so it's possible to embed images in text content if we'd like.

Reply:
@cammellos you mean unfurl a link that leads to an image? or place an inline image next to regular text? I would say it's not something we want, images should always be placed separately as their own message bubble in chat.

Reply:
Yes, that's what I meant (regarding markdown, not the URL, that's a protocol thing), thanks for the reply.

Reply:
this is a temporary implementation, there will be a multihash instead ipfs hash, we don't want to send any web2 links or images,  multihash might be used for ipfs or swarm or our own storage later